tools/online_editor/src/editor_widget.ts

```diff
@@ -83,19 +83,19 @@ function tabTitleFromURL(url: string): string {
 }
 
 class EditorPaneWidget extends Widget {
-  #auto_compile = true;
+  auto_compile = true;
   #style = "fluent";
   #editor_documents: Map<string, ModelAndViewState>;
   #editor: monaco.editor.IStandaloneCodeEditor | null;
-  #keystroke_timeout_handle: number | undefined;
+  #keystroke_timeout_handle: any | undefined;
```

Comment:
> Unexpected any. Specify a different type


I guess `any | undefined `  doesn't really make sense.  It's just `any`
Maybe add a comment why this is any.

Reply:
Every computer I run the linter/syntax checker on seems to produce different warnings right now :-( Where did you see this warning?

`any` since Typescript returns a `number`, while nodeJS returns a `Timeout` object and for some reason some tools insist on parsing as nodeJS and others as plain typescript.

Reply:
`any | undefined` is fine: Tools do not like the `any` with or without the undefined :-)

Reply:
I will drop this: Looks like it is caused by the (unused!) @types/node module being installed locally :-/

Reply:
> Where did you see this warning?

Github action.  the warning is show when you look at the diff on github.